packages/aws-cdk/lib/api/aws-auth/awscli-compatible.ts

```diff
@@ -156,6 +162,15 @@ function hasEcsCredentials(): boolean {
   return (AWS.ECSCredentials.prototype as any).isConfiguredForEcsCredentials();
 }
 
+/**
+ * Return whether it looks like we'll have EKS credentials available
+ * No check like hasEcsCredentials available, so have to implement our own.
+ * @see https://github.com/aws/aws-sdk-js/blob/3ccfd94da07234ae87037f55c138392f38b6881d/lib/credentials/token_file_web_identity_credentials.js#L59
+ */
+function hasEksCredentials(): boolean {
```

Comment:
Maybe `hasWebIdentityCredentials`, with a comment saying this is what EKS containers use as well?

Reply:
@markus7811 You resolved this but didn't apply/comment on my suggestion. Intentional?